src/components/HbLoadingSpinner.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { render } from "src/utils/rtl";
+import { HbLoadingSpinner, HbSpinnerProvider } from "./HbLoadingSpinner";
+
+describe("HbLoadingSpinner", () => {
+  it("picks a quip", async () => {
+    // When we render normally
+    const r = await render(
+      <HbSpinnerProvider quips={["*"]}>
+        <HbLoadingSpinner />
+      </HbSpinnerProvider>,
+      { omitBeamContext: true },
```

Comment:
**question:** Why do we use `omitBeamContext` in all these tests?

Reply:
Artifact from when Beam provided the Provider.